app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def mood_query
     user_moods.joins(:mood).select(:name).group(:name).count
   end
 
+  def mood_over_time
+    user_moods.joins(:mood).group_by_day(:created_at).sum(:rating)
+  end
+
```

Comment:
group_by_day is that an Active Record method?

Reply:
Also, do we need to sum if there is only one rating per day or could we just select the rating?